release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -379,6 +402,45 @@ The Node Feature Discovery (NFD) Operator detects hardware features available on
 
 You can use the new `APIRequestCount` API to track what is using the deprecated APIs. This allows you to plan whether any actions are required in order to upgrade to the next release.
 
+[id="ocp-4-8-monitoring-stack-dependency-version-updates"]
+==== Version updates to monitoring stack components and dependencies
+
+{product-title} 4.8 includes version updates to the following monitoring stack components and dependencies:
+
+* The Prometheus Operator is now on version 0.48.1.
+* Prometheus is now on version 2.26.1.
+* The `node-exporter` agent is now on version 1.1.2.
+* Thanos is now on version 0.20.2.
+* Grafana is now on version 7.5.5.
+
+[id="ocp-4-8-monitoring-kube-state-metrics-v2.0.0-rc.1-upgrade"]
+==== kube-state-metrics upgraded to version v2.0.0-rc.1
+
+`kube-state-metrics` is upgraded to version v2.0.0-rc.1. The following changes include:
+
+* The `--labels-metric-allow-list` is renamed to `--metric-labels-allowlist`.
+* Deprecated Kubernetes APIs are removed.
+* `go.mod` dependencies are updated.
+* Vender folder is removed.
+* `k8s.gcr.io/kube-state-metrics/kube-state-metrics` is now the authoritative registry. Location on `\Quay.io` will no longer be updated. Images that were previously pushed will remain to avoid breaking existing deployments.
```

Comment:
`\Quay.io`
should be
`Quay.io`

Reply:
Updated the `kube-state-metrics` section to only include metrics that were removed in 2.0.0 per Slack conversation with @simonpasquier